configs/stash-cache/xrootd/xrootd-stash-cache.cfg

```diff
@@ -15,10 +15,12 @@
 # https://opensciencegrid.org/docs/data/stashcache/overview/
 #
 
-
 # A few extra configuration settings that have no good other location.
+set EnableHttp=1
```

Comment:
Where does this get used?

Reply:
https://github.com/opensciencegrid/Software-Redhat/blob/trunk/osg-xrootd/osg/50-osg-http.cfg